opentreemap/exporter/tasks.py

```diff
@@ -24,6 +27,8 @@
 from exporter.user import write_users
 from exporter.util import sanitize_unicode_record
 
+_UDFC_FIELDS = tuple(['udf:' + name for name in UDFC_NAMES])
```

Comment:
I do not like the implicit intermediate generator expression that is created implicitly by this syntax in a way that could confuse devs. I try to avoid using it. I also noticed it's slightly slower.
```
~$ time python test.py 
tuple((x + 1) for x in xrange(1000000))

real    0m0.152s
user    0m0.136s
sys     0m0.016s

~$ time python test.py 
tuple([(x + 1) for x in xrange(1000000)])

real    0m0.147s
user    0m0.118s
sys     0m0.028s
```

The stacktrace indicates that it goes through another module for the intermediate generator expression. Interesting thing to note: while the code was nearly the same speed, look how much longer the stack trace took!
```
~$ time python -m trace --listfuncs test.py 
tuple((x + 1) for x in xrange(1000000))

functions called:
filename: /usr/lib/python2.7/trace.py, modulename: trace, funcname: _unsettrace
filename: <string>, modulename: <string>, funcname: <genexpr>
filename: <string>, modulename: <string>, funcname: <module>
filename: test.py, modulename: test, funcname: <module>

real    0m3.799s
user    0m3.785s
sys     0m0.016s

~$ time python -m trace --listfuncs test.py 
tuple([(x + 1) for x in xrange(1000000)])

functions called:
filename: /usr/lib/python2.7/trace.py, modulename: trace, funcname: _unsettrace
filename: <string>, modulename: <string>, funcname: <module>
filename: test.py, modulename: test, funcname: <module>

real    0m0.216s
user    0m0.200s
sys     0m0.019s

```